tests/unit/test_persist_state.c

```diff
@@ -77,7 +77,7 @@ Test(persist_state, test_persist_state_open_success_on_invalid_file)
   unlink(persist_file);
 
   fd = open(persist_file, O_CREAT | O_RDWR, 0777);
-  write(fd, "aaa", 3);
+  cr_assert(write(fd, "aaa", 3) == 3, "Write error on invalid persist file");
```

Comment:
I'd rather you capture the return value into a variable, and then `cr_assert()` on that.

Also makes it easier to set watchpoints on the variable in gdb, etc.